catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java

```diff
@@ -93,6 +93,14 @@ public void testVariableBricks() {
 		assertEquals("Variable has the wrong value after stage", 42.0,
 				dataContainer.getUserVariable("p2", sprite).getValue());
 
+		solo.goBack();
+		solo.goBack();
+		solo.clickOnText("p2");
+		solo.clickOnText("p1");
+		solo.clickOnText("p1");
+		solo.clickOnText(solo.getString(R.string.brick_variable_spinner_create_new_variable));
+		solo.clickOnText(solo.getString(R.string.cancel_button));
+		solo.clickOnText("p1");
 	}
```

Comment:
Would be nice if you could do that in a own method, f.e called `testCreatingNewVariableCancelling()`.
In this way your sourcecode gets commented with the method-name and everyone knows what this code does.
Furthermore use some `assert*` to check the correct behavior of the app.